src/components/insurance/ProductDialog.vue

```diff
@@ -67,24 +70,26 @@ export default {
       }
     },
     title () {
```

Comment:
Why do we need this title()? el-dialog does not support direct this.$t('insurance.product.downloadApp') injection?

Reply:
This wizard is going to be configurated based on product type. So this title will be choosen based on product type.